docs/data/data-grid/joy-ui/GridJoyUISlots.js

```diff
@@ -3,8 +3,8 @@ import Box from '@mui/material/Box';
 import { DataGrid, GridToolbar, GridActionsCellItem } from '@mui/x-data-grid';
 import { unstable_joySlots } from '@mui/x-data-grid/joy';
 import {
-  experimental_extendTheme as materialExtendTheme,
-  Experimental_CssVarsProvider as MaterialCssVarsProvider,
+  createTheme,
+  ThemeProvider as MaterialThemeProvider,
```

Comment:
We don't need to use the `experimental_extendTheme` in this demo.
This makes it easier to test with `@mui/material` v6 because the import path is the same for both v5 and v6